<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>paramMenu</class>
 <widget class="QWidget" name="paramMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_Titre">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Parametre</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Calibrage">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Calibrage</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Wait">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>220</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
